styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,74 @@
 .footer{
-  
-}
\ No newline at end of file
+  .accordionTitle h6{
+    color: $white;
+    font-size: 1.3rem;
+  }
+  ul{
+    list-style: none;
+    @extend .no-padding;
```

Comment:
I propose we use `padding: 0` and `margin: 0` here in favor of the class extension. In addition to being `sass-lint` compliant (placeholders are preferred for extension), we will save a few bytes in text, but more importantly, gain a more flexible design by not being coupled to those classes for simple CSS properties.

`@extend` is fine but we should be more conservative/tactical with it, and if we are to use those classes, we should apply the classes on the elements directly instead of extending it here. Also, because we can't extend classes within media queries, the use of `@extend` is limited, as is seen below in numerous `padding` resets. It will be more maintainable and consistent to just set the properties.